tests/returntypes/test_new_qscript.py

```diff
@@ -362,8 +359,8 @@ def test_broadcasting_multi(self, measurement, expected, shots):
         if shots is None and measurement.return_type is qml.measurements.Sample:
             pytest.skip("Sample doesn't support analytic computations.")
 
-        if measurement.return_type is qml.measurements.State:
-            pytest.skip("State does not support multiple measurements")
+        if measurement.return_type in {State, MutualInfo, VnEntropy}:
+            pytest.skip("Multiple measurements not supported.")
```

Comment:
I think we allow multiple measurements for `mutual_info` and `vn_entropy`. What error do you get here?

Reply:
The skip message was not clear. An error is raised because we are still using parameter broadcasting.

Changed the skip message